spec/features/course/achievement_management_spec.rb

```diff
@@ -59,36 +59,60 @@
                                                           'destroy.success'))
       end
 
-      scenario 'I can award a manually-awarded achievement to a student' do
-        manual_achievement = create(:course_achievement, course: course)
-        auto_achievement = create(:course_achievement, course: course)
-        create(:course_condition_achievement, course: course, conditional: auto_achievement)
+      context 'manually-awarded achievements' do
```

Comment:
Suggest we retain the original message, I think it is more readable (especially for feature specs) to other people trying to figure out what this test is trying to do.

Reply:
Okay, I'll change this.